File: src/main/java/com/rustem/simplecrudapplication/service/CustomerService.java
package com.rustem.simplecrudapplication.service;

import com.rustem.simplecrudapplication.model.Customer;

import java.util.List;

public interface CustomerService {

    Customer getById(Long id);

    void save(Customer customer);

    void delete(Long id);

    List<Customer> getAll();
}

File: src/main/java/com/rustem/simplecrudapplication/service/CustomerService.kt
package com.rustem.simplecrudapplication.service

import com.rustem.simplecrudapplication.model.Customer

interface CustomerService {
    fun getById(id: Long): Customer?
    fun save(customer: Customer)
    fun update(customer: Customer)
    fun delete(id: Long)
    fun getAll(): MutableList<Customer?>
}